### PR TITLE
Authenticated screens

### PR DESCRIPTION
**Pull request by zikowang**
Sorry in advance @luwol03, this will be the last BIG PR >.<
Login (self hosted)
image
Login (vocascan server)
image

Sign Up (self hosted)
image
Sign Up (vocascan server)
image

FirstLogin Screen
image

AddVocab Screen
image

Top Nav
image

SideNav
image

### CODE REVIEW

**Comment by luwol03** [comment]
This is not a problem. Now the linting errors are finally gone and we can develop properly. The screens look great. 
Tomorrow I'll fix the axios stuff.

**Comment by noctera** [comment]
Thanks for fixing and redesigning the screens. They look amazing.

File: src/screens/AddLanguagePackage/AddLanguagePackage.jsx [hunk 1]
```diff
@@ -70,18 +54,17 @@ function AddLanguagePackage(props) {
   ));
 
   return (
-    <Box className={classes.lngpckg}>
-      <TextField
+    <Box className="lngpckg">
```
Comment (luwol03) [inline src/screens/AddLanguagePackage/AddLanguagePackage.jsx:57]: Also here, replace box with div

File: src/App.jsx [hunk 2]
```diff
@@ -33,7 +31,7 @@ function App() {
   } else {
     return (
       <HashRouter>
-        <Box className={classes.root}>
+        <Box className="root">
```
Comment (luwol03) [inline src/App.jsx:34]: Could you replace the mui box component with a div?

File: src/Components/Nav/TopNav.jsx [hunk 3]
```diff
@@ -18,6 +18,7 @@ function TopNav() {
   }
 
   const username = useSelector((state) => state.login.user.username);
+  console.log(username);
```
Comment (luwol03) [inline src/Components/Nav/TopNav.jsx:21]: could you remove the console.log?

File: src/screens/AddVocab/AddVocab.jsx [hunk 4]
```diff
@@ -0,0 +1,197 @@
...
+    ));
+  } else {
+    groupDropdownItems = <MenuItem value="">no groups</MenuItem>;
+  }
+
+  return (
+    <Box className="add-vocab-form">
+      <h1 className="heading">Add vocabulary</h1>
+
+      <Box className="dropdowns">
```
Comment (luwol03) [inline src/screens/AddVocab/AddVocab.jsx:111]: replace with div

**Review by noctera — APPROVED** [review]

File: src/screens/FirstStartup/FirstStartup.jsx [hunk 5]
```diff
@@ -2,14 +2,18 @@ import React from "react";
...
 function FirstStartup(props) {
   const isFirstLogin = useSelector((state) => state.login.firstLogin);
 
+  if (!isFirstLogin) {
+    return null;
+  }
+
   return (
-    <Box className={isFirstLogin ? "first-startup-visible" : "first-startup-invisible"}>
+    <Box className="first-startup-visible">
```
Comment (luwol03) [inline src/screens/FirstStartup/FirstStartup.jsx:16]: Replace with div

**Review by luwol03 — CHANGES_REQUESTED** [review]
Could you replace the mui box component with divs?

File: src/screens/Login/Login.jsx [hunk 6]
```diff
@@ -37,7 +44,7 @@ const Login = (props) => {
       },
     };
     axios
-      .post(serverAddress + "/api/signIn", body, config)
+      .post(`${serverAddress} + /api/signIn`, body, config)
```
Comment (luwol03) [inline src/screens/Login/Login.jsx:47]: this sould be
```js
.post(`${serverAddress}/api/signIn`, body, config)
```
but I will move this into an seperate file soon.

File: src/screens/SelectionScreen/SelectionScreen.jsx [hunk 7]
```diff
@@ -23,6 +23,11 @@ function SelectionScreen(props) {
   function handleSubmit() {
```
Comment (luwol03) [inline src/screens/SelectionScreen/SelectionScreen.jsx:23]: could we rename this function? or could we move it into one function with a parameter?

**Review by luwol03 — CHANGES_REQUESTED** [review]

File: src/screens/Login/Login.jsx (hunk 6 above)
Comment (zikowang) [reply src/screens/Login/Login.jsx:47]: oh yeah

File: src/screens/SelectionScreen/SelectionScreen.jsx (hunk 7 above)
Comment (zikowang) [reply src/screens/SelectionScreen/SelectionScreen.jsx]: One fn with param.

**Review by luwol03 — APPROVED** [review]

**Pull request merged**